core/next.config.ts

```diff
@@ -38,7 +39,10 @@ async function writeSettingsToBuildConfig() {
     locales: data.site.settings?.locales,
     urls: {
       ...data.site.settings?.url,
-      cdnUrl: process.env.NEXT_PUBLIC_BIGCOMMERCE_CDN_HOSTNAME ?? data.site.settings?.url.cdnUrl,
+      cdnUrls: (process.env.NEXT_PUBLIC_BIGCOMMERCE_CDN_HOSTNAME
+        ? process.env.NEXT_PUBLIC_BIGCOMMERCE_CDN_HOSTNAME.split(',').map((s) => s.trim())
+        : [data.site.settings?.url.cdnUrl]
+      ).filter((url): url is string => !!url),
```

Comment:
🍹 Just to make this a bit cleaner, can we extract transforming the `NEXT_PUBLIC_BIGCOMMERCE_CDN_HOSTNAME` before the return so it's something like `manualCdnUrls.length ? manualCdnUrls : [data.site.settings?.url.cdnUrl]`?